types/angular/jqlite.d.ts

```diff
@@ -651,6 +651,7 @@ interface JQuery {
 
     // Undocumented
     length: number;
+    [index: number]: TElement;
```

Comment:
This actually causes a conflict with `@types/jquery` v2.

`error TS2375: Duplicate number index signature.`